lib/ioc-rc

```diff
@@ -360,7 +360,7 @@ __stop_jail () {
     echo "* Stopping $fulluuid ($tag)"
 
     echo -n "  + Running pre-stop"
-    echo "$exec_prestop" | sh
+    eval "$exec_prestop"
```

Comment:
eval feels dirty, but maybe not as dirty as piping to sh? At least some of the base FreeBSD rc.d scripts use eval. Didn't see any pipes to sh in there, so eval might be a better accepted community practice (in absence of the bash style word splitting helper macros).